orchestration/globus/transfer.py

```diff
@@ -115,7 +115,7 @@ def start_transfer(
             relative_path = item.relative_to(source_path.parent)
             tdata.add_item(str(item), os.path.join(dest_path, str(relative_path)))
     else:
-        tdata.add_item(source_path, dest_path)
+        tdata.add_item(str(source_path), str(dest_path))
```

Comment:
If the issue is that the variables are `pathlin.Path` instances, then can we also change the type hints for the variables in the function call?

Reply:
oh wait...

the typing is correct  on the function definition, it's that we reinitizlize source_path.

dest_path probably comes in as a str and doesn't need to be coerced